apps/web/app/liquidity/layout.tsx

```diff
@@ -7,22 +7,22 @@ type Props = Readonly<{
 
 export const metadata: Metadata = {
   title:
-    "Provide Liquidity on MIRA AMM | Earn Rewards with MIRA Liquidity Pools",
+    "Provide Liquidity on MICROCHAIN | Earn Rewards with MICROCHAIN Liquidity Pools",
```

Comment:
AMM is Automated Market Makers.
```suggestion
    "Provide Liquidity on MICROCHAIN AMM | Earn Rewards with MICROCHAIN Liquidity Pools",
```
Do we need to remove that?

Reply:
Lets keep AMM, just replace MIRA